students/grzegorz_kowalik/lesson_03_functions/pow.py

```diff
@@ -0,0 +1,11 @@
+def pow(x, a):
+    if a < 0:
```

Comment:
w treści zadania (jeżeli jest to negative exponent) nie było ograniczenia, że nie możesz użyć operatora `**` ;)